azext_iot/_help.py

```diff
@@ -295,6 +295,30 @@
         --auth-type identity --identity {managed_identity_resource_id}
 """
 
+helps[
+    "iot hub device-identity generate-mqtt-credentials"
+] = """
+    type: command
+    short-summary: Generate user credentials for MQTT Device sign in.
```

Comment:
Should be along the lines of "Generate mqtt credentials for device or module connections."

Reply:
lets also update the example verbiage